pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -977,6 +980,50 @@ public PersistentOfflineTopicStats getBacklog(
         return internalGetBacklog(authoritative);
     }
 
+    @GET
+    @Path("/{tenant}/{namespace}/{topic}/backlogQuotaMap")
```

Comment:
```suggestion
    @Path("/{tenant}/{namespace}/{topic}/backlogQuota")
```

Reply:
I think it is best to keep consistent with the namespace.

Reply:
Ok, make sense.